measuring-client/build.gradle

```diff
@@ -56,7 +56,7 @@ android {
 
     // Load credentials
     def properties = new Properties()
-    properties.load(new FileInputStream("local.properties"))
+    file("local.properties").withInputStream { properties.load(it) }
```

Comment:
This groovy method should auto-close the stream, see it's [documentation](http://docs.groovy-lang.org/latest/html/groovy-jdk/java/io/File.html):
"Create a new InputStream for this file and passes it into the closure. This method ensures the stream is closed after the closure returns."

Reply:
Didn't find or see anything else used in Gradle.